test/visitors/test_process_connectivity_visitor.py

```diff
@@ -0,0 +1,167 @@
+# type: ignore
+from bpmncwpverify.builder.process_builder import ProcessBuilder
+import pytest
+from bpmncwpverify.core.bpmn import Bpmn, EndEvent, StartEvent
+from bpmncwpverify.visitors.process_connectivity_visitor import (
+    ProcessConnectivityVisitor,
+)
+import xml.etree.ElementTree as ET
+
+
+def test_process_connectivity(mocker):
```

Comment:
We need better names for the tests. What aspect of connectivity is this test testing? I use behavior driven development naming conventions with _given_, _when_, and _then_ in the test name. Given: certain types of input. When: the interface being called. Then: what should happen. See `test_state.py`.